source/_posts/setup-tailwindcss_parcel_npm.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Setup Tailwindcss with Parcel.js or npm.
+date: 2018-07-19
```

Comment:
Yesterda's date? I think it's better to edit the date to the merge date or the intended publishing date